guides/source/command_line.md

```diff
@@ -326,7 +326,7 @@ With the `helper` method it is possible to access Rails and your application's h
 
 ### `rails dbconsole`
 
-`rails dbconsole` figures out which database you're using and drops you into whichever command line interface you would use with it (and figures out the command line parameters to give to it, too!). It supports MySQL, PostgreSQL and SQLite3.
+`rails dbconsole` figures out which database you're using and drops you into whichever command line interface you would use with it (and figures out the command line parameters to give to it, too!). It supports MySQL/MariaDB, PostgreSQL and SQLite3.
```

Comment:
There are other MySQL and PostgreSQL compatible RDBMS that use the same CLI, too, but we don't list them all here. I think we don't need to repeat /MariaDB here.